user_docs/en/changes.md

```diff
@@ -37,6 +37,9 @@ Add-ons will need to be re-tested and have their manifest updated.
   * The changelog should document changes between previous and latest add-on versions.
 * Updated components
   * Licensecheck has been updated to 2025.1 (#18728, @bramd)
+* Added the `utils.ctypesUtils` module, which implements a convenience decorator (`dllFunc`) to create ctypes function wrappers compatible with static type checkers.
+It is mainly used within the `winBindings` module that was introduced in NVDA 2025.3.
+Consult the developer guide for more information. (#18534, @LeonarddeR)
```

Comment:
It is actually in the coding standards. Should I move it?
```suggestion
Consult our coding standards for more information. (#18534, @LeonarddeR)
```

Reply:
Yes, I think this makes more sense in the developer guide rather than coding standards